common/blobstore/retryableClient_test.go

```diff
@@ -0,0 +1,156 @@
+// The MIT License (MIT)
```

Comment:
let's rename the files to snake_case as we touch

Reply:
Okay! renaming.